lib/c/build/install-musl.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
```

Comment:
Probably worth setting `SYSROOT` (or `SYMBOLICA_HOME`) in this script and using that throughout (or pending my other comment, relying on it already being set in the current environment.

Reply:
I'd rather leave this env var stuff alone for now, but maybe soon as a separate PR. I did have something like this in the past and it was pretty annoying.